### PR TITLE
now actually download the new models

### PR DESCRIPTION
**Pull request by alinelena**

### CODE REVIEW

File: mace/calculators/foundations_models.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def download_mace_mp_checkpoint(model: Union[str, Path] = None) -> str:
 
     checkpoint_url = (
         urls.get(model, urls["medium"])
-        if model in (None, "small", "medium", "large")
+        if model in (None, "small", "medium", "large", "small", "small-0b", "medium-0b", "small-0b2", "medium-0b2", "large-0b2")
```
Comment (oerc0122) [inline mace/calculators/foundations_models.py:45]:
```suggestion
        if model in (None, "small", "medium", "large", "small-0b", "medium-0b", "small-0b2", "medium-0b2", "large-0b2")
```

Comment (oerc0122) [reply mace/calculators/foundations_models.py:45]: Do we want a top-level constant of these rather than one here repeated later?

Comment (ElliottKasoar) [reply mace/calculators/foundations_models.py]: Or maybe `if model is None or model in urls`?

`urls` could be the top-level constant, to reuse it.

**Comment by alinelena** [comment]
best will be high level constants

**Pull request merged**